scripts/prepare

```diff
@@ -39,6 +39,13 @@ for f in packages/*; do
 			if [[ "${split_crds}" == "true" ]]; then
 				./scripts/prepare-crds ${f}
 			fi
+			# Add provides GVR annotation, this lets other charts require it
+			providesGVR="$(cat ${f}/package.yaml | yq r - providesGVR)"
+			if ! [[ -z ${providesGVR} ]]; then
+				if [[ -z "$(yq r ${f}/charts/Chart.yaml 'annotations[catalog.cattle.io/provides-gvr]')" ]]; then
```

Comment:
hmm, do we need the `provides-gvr` in the `package.yaml` anymore? 

We initially kept the `provides-gvr` annotation value in the `package.yaml` since the same value needed to exist in both the original chart and the CRD chart. But now we only use that annotation in the original chart, so maybe it should just belong in the patch.

Reply:
It's no longer necessary, but it's easier to adjust in package.yaml than adjusting everyone's annotations in their patches. We could fix in future and switch it everywhere.